hazelcast/src/test/java/com/hazelcast/quorum/PartitionedCluster.java

```diff
@@ -200,21 +201,21 @@ private void splitCluster() {
         cm5.block(n2.address);
         cm5.block(n3.address);
 
-        n4.clusterService.suspectMember(n1.getLocalMember(), null, true);
-        n4.clusterService.suspectMember(n2.getLocalMember(), null, true);
-        n4.clusterService.suspectMember(n3.getLocalMember(), null, true);
+        suspectMember(n4, n1);
```

Comment:
You should also update https://github.com/hazelcast/hazelcast-enterprise/blob/master/hazelcast-enterprise/src/test/java/com/hazelcast/quorum/NativeMemPartitionedCluster.java in EE repo after this gets merged.